client/src/app/pages/applications/components/application-detail-drawer/application-detail-drawer-assessment.tsx

```diff
@@ -134,3 +147,17 @@ export const ApplicationDetailDrawerAssessment: React.FC<
     />
   );
 };
+const ArchetypeLabels: React.FC<{ archetypeRefs?: Ref[] }> = ({
+  archetypeRefs,
+}) =>
+  (archetypeRefs?.length ?? 0) === 0 ? null : (
```

Comment:
With the length wrapper already in the description list, `archetypeRefs` should be required and this step can be skipped.

...or push the "None" text into this component